modules/installation-osp-kuryr-config-yaml.adoc

```diff
@@ -0,0 +1,85 @@
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-kuryr-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for OpenStack with Kuryr SDN
+
+To deploy with Kuryr SDN instead of the default OpenShift SDN, you simply need
+to modify the `install-config.yaml` file to include `Kuryr` as the desired
+`networking.networkType` and proceed with the same steps as with the default
+OpenShift SDN. This sample `install-config.yaml` demonstrates all of the
+possible {rh-openstack-first} customization options.
+
+[IMPORTANT]
+This sample file is provided for reference only. You must obtain your
+`install-config.yaml` file by using the installation program.
+
+[source, yaml]
+----
+apiVersion: v1
+baseDomain: example.com
+clusterID: os-test
+controlPlane:
+  name: master
+  platform: {}
+  replicas: 3
+compute:
+- name: worker
+  platform:
+    openstack:
+      type: ml.large
+  replicas: 3
+metadata:
+  name: example
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineCIDR: 10.0.0.0/16
+  serviceNetwork:
+  - 172.30.0.0/16
+  networkType: Kuryr
+platform:
+  openstack:
+    region: region1
+    cloud: mycloud
+    externalNetwork: external
+    computeFlavor: m1.xlarge
+    lbFloatingIP: 128.0.0.1
+    trunkSupport: true
+    octaviaSupport: true
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+----
+
+
+[NOTE]
+Both `trunkSupport` and `octaviaSupport` are automatically discovered by the
+installer, so there is no need to set them. But if your environment does not
+meet both requirements Kuryr SDN will not properly work, as trunks are needed
+to connect the pods to the OpenStack network and Octavia to create the
+OpenShift services.
+
+[id="kuryr-limitations"]
+= Known limitations of installing with Kuryr SDN
+
+There are known limitations when using Kuryr SDN:
+
+* There is an amphora load balancer VM being deployed per OpenShift service with
+  the default Octavia load balancer driver (amphora driver). If the environment
+  is resource constrained it could be a problem to create a large amount of
+  services.
+* Depending on the Octavia version, UDP listeners are not supported. This means
+  that OpenShift UDP services are not supported.
+* There is a known limitation of Octavia not supporting listeners on different
+  protocols (e.g., UDP and TCP) on the same port. Thus services exposing the
+  same port for different protocols are not supported.
+* Due to the above UDP limitations of Octavia, Kuryr is forcing pods to use TCP
+  for DNS resolution (`use-vc` option at `resolv.conf`). This may be a problem
+  for pods running Go applications compiled with `CGO_DEBUG` flag disabled as
+  that forces to use the `go` resolver that is only using UDP and is not
+  considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a
```

Comment:
"This may be a problem for Pods running Go applications compiled with the `CGO_DEBUG` flag disabled, as that uses the `go` resolver that only leverages UDP and is not considering the `use-vc` option added by Kuryr to the `resolv.conf`.